backend/src/graphql/resolvers/mutations.js

```diff
@@ -1,5 +1,26 @@
 const User = require('../../models/user');
+const { attemptLogin, attemptLogout } = require('../../auth');
 
 module.exports = {
```

Comment:
Personal preference: I like using exporting individually to avoid the indentation.